forespoersel-besvart/src/test/kotlin/no/nav/helsearbeidsgiver/inntektsmelding/forespoerselbesvart/AvsenderSystemLoeserTest.kt

```diff
@@ -0,0 +1,111 @@
+package no.nav.helsearbeidsgiver.inntektsmelding.forespoerselbesvart
+
+import io.kotest.core.spec.style.FunSpec
+import io.kotest.matchers.ints.shouldBeExactly
+import io.kotest.matchers.shouldBe
+import io.mockk.clearAllMocks
+import io.mockk.every
+import io.mockk.mockk
+import no.nav.helse.rapids_rivers.testsupport.TestRapid
+import no.nav.helsearbeidsgiver.felles.AvsenderSystemData
+import no.nav.helsearbeidsgiver.felles.BehovType
+import no.nav.helsearbeidsgiver.felles.DataFelt
+import no.nav.helsearbeidsgiver.felles.EventName
+import no.nav.helsearbeidsgiver.felles.Key
+import no.nav.helsearbeidsgiver.felles.json.toJson
+import no.nav.helsearbeidsgiver.felles.json.toJsonNode
+import no.nav.helsearbeidsgiver.felles.rapidsrivers.model.Data
+import no.nav.helsearbeidsgiver.felles.rapidsrivers.model.Fail
+import no.nav.helsearbeidsgiver.felles.test.json.toDomeneMessage
+import no.nav.helsearbeidsgiver.felles.test.rapidsrivers.firstMessage
+import no.nav.helsearbeidsgiver.felles.test.rapidsrivers.sendJson
+import no.nav.helsearbeidsgiver.felles.utils.randomUuid
+import no.nav.helsearbeidsgiver.inntektsmelding.forespoerselbesvart.spinn.FIKK_SVAR_MED_RESPONSE_STATUS
+import no.nav.helsearbeidsgiver.inntektsmelding.forespoerselbesvart.spinn.SpinnApiException
+import no.nav.helsearbeidsgiver.inntektsmelding.forespoerselbesvart.spinn.SpinnKlient
+import no.nav.helsearbeidsgiver.utils.json.toJson
+import no.nav.helsearbeidsgiver.utils.test.mock.mockStatic
+import java.util.UUID
+
+val avsenderSystemData = AvsenderSystemData(
+    avsenderSystemNavn = "NAV_NO",
+    avsenderSystemVersjon = "1.63",
```

Comment:
Vil det være teoretisk mulig å motta en avsenderSystemData med tomme verdier for versjon el.l.? Altså at vi får 200 OK men noen verdier er ikke satt. Mulig dette ikke er nødvendig, men vurdere å ha et testcase for det også, for å sjekke at det håndteres ok?

Reply:
Kan også være at du allerede har håndtert dette med eksisterende kode, men at jeg bare har oversett det

Reply:
Jeg ser nå jeg ikke har laget test for det caset så kan lage en test for det 👍 
Jeg vet ikke om det er praktisk mulig men pga kontrakten så er det teoretisk mulig.